server/views/pages/search/probationSearch/probationSearch.njk

```diff
@@ -0,0 +1,245 @@
+{% set pageId = "probation-search-page" %}
+{% set backLinkHref = backLink %}
+{% set noJsBackLink = true %}
+
+{% macro offenderName(offenderName, crn, prisonerNumber, licenceId, comName, index) %}
+    <div class="search-offender-name">
+        {% set link = "/licence/create/id/" + licenceId + "/check-your-answers" if licenceId else "/licence/create/nomisId/" + prisonerNumber + "/confirm"%}
+        {% if comName %}
+            <a id="name-button-{{ index }}" href="{{ link }}" class="govuk-link govuk-heading-s govuk-!-padding-left-0 govuk-!-margin-bottom-0">{{ offenderName }}</a>
+        {% else %}
+            <span id="name-button-{{ index }}" class="govuk-heading-s govuk-!-padding-left-0 govuk-!-margin-bottom-0">{{ offenderName }}</span>
+        {% endif %}
+        <div class="govuk-hint govuk-!-font-size-16">CRN: {{ crn }}</div>
+    </div>
+{% endmacro %}
+
+{% macro displayResults(countText, results, setPeopleInPrisonTab) %}
+    {% if results.length > 0 %}
+        {% if setPeopleInPrisonTab %}
+            {% set caption = "In Prison Search Results" %}
+            <h2 class="govuk-heading-l" id = "tab-heading-prison">{{ countText }}</h2>
+        {% else %}
+            {% set caption = "On Probation Search Results" %}
+            <h2 class="govuk-heading-l" id = "tab-heading-probation">{{ countText }}</h2>
+        {% endif %}
+        <form method="GET">
+            {{ govukTable({
+                attributes: {
+                    'data-module': 'moj-sortable-table'
+                },
+                caption: caption,
+                captionClasses: "govuk-visually-hidden",
+                head: [
+                    {
+                        text: "Name",
+                        attributes: {
+                            "aria-sort": "ascending"
+                        }
+                    },
+                    {
+                        text: "Licence type"
+                    },
+                    {
+                        text: "Probation practitioner",
+                        attributes: {
+                            "aria-sort": "none"
+                        }
+                    },
+                    {
+                        text: "Team name",
+                        attributes: {
+                            "aria-sort": "none"
+                        }
+                    },
+                    {
+                        text: "Release date",
+                        attributes: {
+                            "aria-sort": "none"
+                        }
+                    },
+                    {
+                        text: "Status",
+                        attributes: {
+                            "aria-sort": "none"
+                        }
+                    }
+                ],
+                rows: results
+            }) }}
+        </form>
+    {% else %}
+        {% if setPeopleInPrisonTab %}
+            <div class="govuk-grid-row">
```

Comment:
can you do something like? 
```
{% if setPeopleInPrisonTab %}
            {% set caption = "In Prison Search Results" %}
            <h2 class="govuk-heading-l" id = "tab-heading-prison">{{ countText }}</h2>
{% else %}
            {% set caption = "On Probation Search Results" %}
            <h2 class="govuk-heading-l" id = "tab-heading-probation">{{ countText }}</h2>
{% endif %}

{% if results.length == 0 %}
  ... render message
{% elseif %}
  ... render rows
{% endif %}
```

Reply:
would be tempted to pass in caption and headingId as separate macro params and get rid of the initial condition too
```
    <h2 class="govuk-heading-l" id = "tab-heading-{{headingId}}">{{ countText }}</h2>
```

You can wrap the params in an object like in javascript if you want the effect of named params

Reply:
Cool, would it be ok if I add these changes to the final PR for the search - got one more to come?

Reply:
sure, added another suggstion around using a filter to pluralise which could also be done as a follow up